src/main/java/it/aboutbits/springboot/emailservice/lib/application/EmailParameter.java

```diff
@@ -0,0 +1,62 @@
+package it.aboutbits.springboot.emailservice.lib.application;
+
+import it.aboutbits.springboot.emailservice.lib.AttachmentReference;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotEmpty;
+import lombok.Builder;
+import lombok.NonNull;
+import lombok.Singular;
+import org.springframework.lang.Nullable;
+
+import java.time.OffsetDateTime;
+import java.util.List;
+import java.util.Set;
+
+@Builder
+public record EmailParameter(
+        @NonNull OffsetDateTime scheduleAt,
```

Comment:
Are the `@NonNull` annotation necessary if we already have the package-info?

Reply:
I'll check.

Reply:
I think so, just to be safe